_posts/2020-06-02-tapping-glass.markdown

```diff
@@ -0,0 +1,127 @@
+and to have my default blank Blender file created so I wouldn't need to navigate my filesystem with screen sharing on.
+The rest was made as part of the live demo.
+
+By the way, that HDRI environment texture is ["Machine Shop 02", by Sergej Majboroda](https://hdrihaven.com/hdri/?c=indoor&h=machine_shop_02), published under a [CC0](https://creativecommons.org/publicdomain/zero/1.0/) license.
+
+I would estimate the presentation took 20 minutes over the 2 parts, leaving only 40 minutes for the speed art.
+At some points, I was worried I wouldn't finish on time.
+I managed to get the render done at a lower resolution before the 1 hour was up.
+I re-rendered it out in 4K later.
+I'm quite satisfied with the result.
+Some others have told me it is photoreal, which makes me very happy.
+
+</div>
+
+# Planning for speed
+{: #planning-for-speed}
+
+{% include collapser.markdown %}
+
+<div>
+
+I came up with the idea for the scene well in advance, and I planned out what I would make so that the result would be good.
+The quick sketch is even in the slides I made.
+
+The requirements for the scene can be broken down into 3 goals:
+
+1. It must not take very long to make. I can't exceed the time limit and I wanted the render to be done in time. This is the strictest requirement.
+2. It should pass. Perhaps not photoreal, but it should look very good.
+3. It should be interesting. There should be something going on in the scene, and you actually want to look at it. Being interesting is a desireable property, perhaps even required, for art.
+
+I did some brainstorming, excluded the ideas that were unlikely to work out, refined the ideas and explored more, and repeated this until I settled on a viable idea.
+
+Every material can benefit from textures, but some materials can look passable without textures, including smooth metal and clean glass, which I ended up using.
+There are plenty of other materials that fit the bill too, including water, fog, mirrors, jelly, and some smooth plastics, so it's not like I used the only options available.
+In fact, in the iteration before what I finally stuck with, I intended to have a jelly-looking tentacle whack the glass cup and break it,
+but then I realized the tentacle would need to be attached to something, and that image was unappealing to me.
+I'm glad I chose to use a metal rod instead, since the issue of where to put the tentacle isn't the only issue I would have with it -
+it's also a more complicated shape, would need rigging, and demand soft body physics to make the scene pass.
+
+Physics simulations tend to be slow, and they would generally be unviable for speed art.
+However, they're also free animation that you don't need to make by hand.
+Breaking glass is a classic interesting visual, and it would mean I only need the small amount of animation to set up the breaking glass, and the rest comes for free.
```

Comment:
```suggestion
Breaking glass is a classic and interesting visual, and it would mean I only need the small amount of animation to set up the breaking glass, and the rest comes for free.
```
Improving sentence flow.

Reply:
This would change the meaning. There's a slight difference in how "classic" is interpreted when it's applied to the already modified noun "interesting visual". What I'm trying to say is that it's an interesting visual, and among interesting visuals it's a classic.

I think I'll keep the original wording.

Reply:
Also, I'm not sure if "classic" is even a normal adjective. The suggested changed version feels a bit wrong. But it works fine for modifying certain nouns like this when placed directly before it.